.changeset/mighty-cycles-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': minor
```

Comment:
Don't think we need to bump this library. It's private so don't think people will depend on these.